lib/node_modules/@stdlib/math/base/special/ellipe/README.md

```diff
@@ -2,7 +2,7 @@
 
 @license Apache-2.0
 
-Copyright (c) 2019 The Stdlib Authors.
+Copyright (c) 2024 The Stdlib Authors.
```

Comment:
```suggestion
Copyright (c) 2019 The Stdlib Authors.
```

Reply:
Had a doubt. Should we change the years of already existing files like this one too ?

Reply:
Yes, so this README.md is not new, it was pre-existent.

Reply:
Also, I've added the newlines, thanks !